src/plugins/directory.js

```diff
@@ -0,0 +1,59 @@
+export const classProperties = {
```

Comment:
Thinking about this again...

most if not all of the plugins have a corresponding babel plugin that is just the same name.

objectRestSpread -> transform-object-rest-spread
asyncGenerators -> transform-async-generators

the only 2 that don't have a corresponding preset now

flow -> preset-flow
react -> preset-react